tests/test_matrix.py

```diff
@@ -278,3 +278,11 @@ def test_multiplyMatrix():
     # assert matPro.__str__() == ""
     """
     pass
+
+
+def test_determinant(mat='[1,2;3,4]'):
+    mat = getTokens(mat)
+    if mat.isSquare():
+        return mat.determinant()
```

Comment:
Use `assert` to verify your function's output.
```suggestion
        assert mat.determinant() == <determinant-value>
```
Also add test cases for matrix of size 1 and 3.
Lines 287-288 can be removed.